Framework/PythonInterface/plugins/algorithms/LRPeakSelection.py

```diff
@@ -1,12 +1,12 @@
-#pylint: disable=no-init,invalid-name
+# pylint: disable=no-init,invalid-name, attribute-defined-outside-init, too-many-instance-attributes
```

Comment:
Was this new disable `attribute-defined-outside-init` needed here? I'd rather leave some warnings in there, than disabling them when not absolutely required. This warning can sometimes highlight code that is not so well organized and declares attributes in strange places.